### PR TITLE
Implement playable item headers

### LINKED ISSUE #52: Implement playable item header
## Motivation
See #47 

Designs:
- [Single podcast episode](https://projects.invisionapp.com/share/EVE2P3L4R#/screens/259676370)
- [Single liturgy item](https://projects.invisionapp.com/share/EVE2P3L4R#/screens/261591805)
- [Single meditation](https://projects.invisionapp.com/share/EVE2P3L4R#/screens/259684185)

### PR DESCRIPTION
**Pull request by brettdh**
# Description

Closes #52.

The layout may need further tweaks as we start to use it and test across more devices.
I'm generally not very confident on how to ensure good layout across devices. Generally
I try to lean on flexbox as much as possible for that, but font sizes are a tricky point in 
this PR especially.

# Screenshots

liturgy
meditation
podcast

### CODE REVIEW

File: src/components/PlayButton.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <Icon name="play" style={styles.playIcon} />
+    <Text style={styles.buttonText}>{text}</Text>
+  </Button>
+);
+
+PlayButton.propTypes = {
+  text: PropTypes.string,
+  style: View.propTypes.style,
+  // passthrough props:
+  // - onPress
```
Comment (CaleCrawford) [inline src/components/PlayButton.js:45]: I like this. Im definitely gonna steal this method in the future.

File: src/components/PlayableItemHeader.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  if (!_.isNull(publishDate)) {
+    strings.push(formatPublishDate(publishDate));
+  }
+  return strings.join(separator);
+}
+
+function screenRelativeWidth(fraction) {
+  const { width } = Dimensions.get('window');
+  return width * fraction;
+}
```
Comment (CaleCrawford) [inline src/components/PlayableItemHeader.js:55]: I like this and I almost used something like it in the series tiles. Should we turn this into a global utility or something like that?

File: src/components/PlayableListCard.js [hunk 3]
```diff
@@ -155,7 +156,7 @@ PlayableListCard.defaultProps = {
   style: {},
   duration: null,
   publishDate: null,
-  formatDuration: m => `${m.asMinutes()} minutes`,
+  formatDuration: formatMinutesString,
```
Comment (CaleCrawford) [inline src/components/PlayableListCard.js:159]: Disregard my comment from the last PR :)

**Review by CaleCrawford — APPROVED** [review]
LGTM. We will have to figure out the styling. Its a bummer that normalize method from the component lib doesn't seem to be working. I like the utility functions that you added.
One thing that isnt a big deal at all is that the patreon images seem to have stopped showing up. I think one of the params is a timestamp key that makes sure people like us aren't hotlinking ;). Once the API is up im sure we can use that.

File: src/components/PlayableItemHeader.js (hunk 2 above)
Comment (brettdh) [reply src/components/PlayableItemHeader.js:55]: Sure; we could do that at the same time as updating the series tiles to use it. However, as you mentioned in another PR for fonts/line height, his might also be something that we should just do once everywhere, for all things like this.

**Pull request merged**